arbnode/batch_poster.go

```diff
@@ -542,11 +542,13 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 
 	if b.daWriter != nil {
 		cert, err := b.daWriter.Store(ctx, sequencerMsg, uint64(time.Now().Add(config.DASRetentionPeriod).Unix()), []byte{}) // b.daWriter will append signature if enabled
-		if err != nil {
-			log.Warn("Unable to batch to DAS, falling back to storing data on chain", "err", err)
+		if errors.Is(err, das.BatchToDasFailed) {
+			log.Warn("err", err)
```

Comment:
I think with the geth logger you need a message string, followed by pairs of `"field-name"` and `field` otherwise it prints an error that there are a mismatched number of fields.

Reply:
Done.